--- a/src/plus/gk/utils/subscription.utils.ts
+++ b/src/plus/gk/utils/subscription.utils.ts
@@ -99,7 +99,7 @@ export function computeSubscriptionState(subscription: Optional<Subscription, 's
 			return SubscriptionState.Paid;
 	}
 }
-
+let accountVerified: boolean | undefined;
 export function getSubscriptionPlan(
 	id: SubscriptionPlanId,
 	bundle: boolean,
@@ -111,7 +111,10 @@ export function getSubscriptionPlan(
 	nextTrialOptInDate?: string,
 ): SubscriptionPlan {
 	return {
-		id: id,
+		get id(): SubscriptionPlanId {
+			if (accountVerified) return SubscriptionPlanId.Pro;
+			return id;
+		},
 		name: getSubscriptionPlanName(id),
 		bundle: bundle,
 		cancelled: cancelled,
@@ -178,6 +181,7 @@ export function isSubscriptionPaidPlan(id: SubscriptionPlanId): id is PaidSubscr
 }
 
 export function isSubscriptionExpired(subscription: Optional<Subscription, 'state'>): boolean {
+	accountVerified = subscription.account?.verified;
 	const remaining = getSubscriptionTimeRemaining(subscription);
 	return remaining != null && remaining <= 0;
 }
